docs/src/contrasts.md

```diff
@@ -0,0 +1,77 @@
+```@meta
+CurrentModule = StatsModels
+```
+
+# Modeling categorical data
+
+To convert categorical data into a numerical representation suitable for
+modeling, `StatsModels` implements a variety of __contrast coding strategies__.
+Each contrast coding strategy maps a categorical vector with $k$ levels onto
+$k-1$ linearly independent model matrix columns.
+
+## How to specify contrast coding
+
+The default contrast coding strategy is `DummyCoding`.  To override this, use
+the `contrasts` argument when constructing a `ModelFrame`:
+
+```julia
+mf = ModelFrame(y ~ 1 + x, df, contrasts = Dict(:x => EffectsCoding()))
```

Comment:
`EffectsCoding` hasn't been mentioned yet. Maybe better list first the supported strategies, and only then say how to use them?